src/kit/LineChart.tsx

```diff
@@ -270,25 +267,6 @@ export const LineChart: React.FC<LineChartProps> = ({
         options={chartOptions}
         xAxis={xAxis}
       />
-      {showLegend && (
-        <div className={css.legendContainer}>
```

Comment:
you can remove CSS related to this customized legend (legendContainer, legend, hideSeries, colorButton)